### DISCUSSION TITLE
“make -j 4” is wrong,why?

### DISCUSSION THREAD

**Discussion by vicLzs** [idaholab/moose#19295] [Q&A Getting Started] [answered]
when i enter this command"make -j 4" ,there is a wrong pop up.
![图片](https://user-images.githubusercontent.com/57779487/140276672-28037a1b-2e69-4f08-a9e9-e3326ed1bbf3.png)

**Comment by loganharbour** [comment]
In the future, please do not create or comment on multiple issues pertaining to the same problem for an initial report.

Transferring to discussion as an issue is not appropriate.

**Comment by loganharbour** [comment, marked as answer]
I see that you are running this on a virtual machine. What kind of resources do you have allocated to it (number of processors, total memory, total disk storage)?

The issue here is likely that the compiler is failing because your VM is running out of resources.
